Dockerfile.dev

```diff
@@ -0,0 +1,17 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.4
+
+WORKDIR /
+
+COPY direct-csi /direct-csi
+COPY CREDITS /licenses/CREDITS
+COPY LICENSE /licenses/LICENSE
+
+COPY centos.repo /etc/yum.repos.d/centos.repo
+
+RUN \
+    curl https://www.centos.org/keys/RPM-GPG-KEY-CentOS-Official -o /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-Official && \
+    mv /etc/yum.repos.d/ubi.repo /etc/yum.repos.d/ubi.repo.old && \
+    microdnf install xfsprogs --nodocs && \
```

Comment:
You should also do `microdnf update` as well to get latest updates if any?

Reply:
It looks like centos is the upstream now, and microdnf update does not always give signed packages. It seems liek the best approach for security reasons is we should use specific package versions when updating.

Reply:
I meant we need to update for RedHat packages, and perhaps install xfsprogs and then remove the CentOS.repo just in case to avoid any additional repos in the container.